Steamworks/src/org/usfirst/frc/team4786/robot/RobotMap.java

```diff
@@ -10,18 +10,47 @@
 	
 	//CANTalon ports, use instead of random numbers
 	//Not Final at ALL!!!
-	public static final int backLeftPort = 15;
-	public static final int backRightPort = 12;
-	public static final int frontLeftPort = 14;
-	public static final int frontRightPort = 13;
+	public static final int backLeftPort = 14;
+	public static final int backRightPort = 13;
+	public static final int frontLeftPort = 15;
+	public static final int frontRightPort = 12;
 	
 	//Scaling used only for open loop drive system
 	public static final double openLoopSpeedScaling = .6;
 	
+	//General PID Constants
+	//public static final double ERROR_CONSTANT = 45;
+	public static final int DRIVETRAIN_ENCODER_CODES_PER_REV = 360;
+	//Wheel Radius measured in feet
+	public static final double WHEEL_RADIUS = 0.25;
+	//Left GearBox PID Constants
+	public static final double LeftP = 0.355 / 7; //35% throttle within 5 rotations of target
```

Comment:
This doesn't need to be fixed now, but when the speed/velocity PID constants are added, these should be renamed to include that they have position information in them (like `LeftPositionP` or something).